x-pack/legacy/plugins/canvas/i18n/functions/function_help.ts

```diff
@@ -151,12 +153,12 @@ type FunctionHelpMap<T> = T extends ExpressionFunction<
 //
 // This type indexes the existing function factories, reverses the union to an
 // intersection, and produces the dictionary of strings.
-type FunctionHelpDict = UnionToIntersection<FunctionHelpMap<CanvasFunction>>;
+type FunctionHelpDict = UnionToIntersection<FunctionHelpMap<AnyExpressionFunctionDefinition>>;
```

Comment:
We had this CanvasFunction type setup so that we would get a type check error if FunctionHelpDict was missing an entry for one of our functions.

Reply:
Good catch! I've put it back.